config.py

```diff
@@ -7,7 +7,11 @@
 DATABASE = os.path.join(os.getcwd(), 'DCC.db')
 SCHEMA = os.path.join(os.getcwd(), 'schema.sql')
 SECRET_KEY = 'development key'
-SERVER_TOKENS = ['your-server-token']
+SERVER_TOKENS = {'server-token':
+                     {'name': 'server name',
+                      'address': 'http://example.com'
+                      }
```

Comment:
The server token is for authentication rather than stable identification or namespacing. Rather than `address`, we should have an `id` field here, that's used to populate the `server_id` database field.